### DISCUSSION TITLE
Can deploy on preview but not on production

### DISCUSSION THREAD

**Discussion by da7a90-backup** [vercel/next.js#50516] [Help]
### Summary

I have a NextJS application containing an edge function that uses the jimp library, specifically the browser build, it doesn't use dynamic code evaluation on runtime but because of Istanbul or a global shim there is always a couple of `evals` and `Function("")` statements in there no matter what I do. I used `unstable_allowDynamic` which worked locally and was able to build for preview environment but prod throws this error everytime:

```
Failed to compile.
2023-05-30T05:27:33.316Z  
2023-05-30T05:27:33.317Z  ./node_modules/jimp/browser/lib/jimp.js
2023-05-30T05:27:33.317Z  Dynamic Code Evaluation

```

### Additional information

I tried multiple solutions with limited success, it seems like the preview environment is not really a reliable way to test for production.

**Comment by dondxniel** [comment]
@da7a90-backup were you able to figure out what the issue is? I'm having the same problem.
